src/app/Fake.Azure.Kudu/Kudu.fs

```diff
@@ -0,0 +1,122 @@
+﻿/// Contains tasks to stage and deploy Azure website and webjobs using source code deployment with Kudu Sync.
+module Fake.Azure.Kudu
+
+open System
+open System.IO
+#if NETSTANDARD
+open System.Net.Http
```

Comment:
Can we always use `System.Net.Http` here?

Reply:
Is it available in full framework? I copied this from the `Fake.Api.Slack` project.